fontforgeexe/fontinfo.c

```diff
@@ -677,12 +677,12 @@ static GTextInfo panserifvar[] = {
 static GTextInfo pantreatment[] = {
     { (unichar_t *) N_("PanoseTreatment|Any"), NULL, 0, 0, (void *) 0, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
     { (unichar_t *) N_("PanoseTreatment|No Fit"), NULL, 0, 0, (void *) 1, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
-    { (unichar_t *) N_("Solid Fill"), NULL, 0, 0, (void *) 2, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
+    { (unichar_t *) N_("Standard Solid Fill"), NULL, 0, 0, (void *) 2, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
```

Comment:
I think it tries to explain the "None". 2 None fill results in the standard fill will be used in the rendering of the font, they would call that black. 3 White results in no fill being used in the rendering and they call that white. FontForge luckily doesn't name black and white, which I think is good as a font and its background can be rendered in any colour.